RInAppMessaging/Classes/Models/Responses/DisplayPermissionResponse.swift

```diff
@@ -1,4 +1,33 @@
-internal struct DisplayPermissionResponse: Decodable {
+internal struct DisplayPermissionResponse: Codable, Equatable {
+
+    private enum CodingKeys: String, CodingKey {
+        case display
+        case performPing
+        case creationTimeMilliseconds
+    }
+
     let display: Bool
     let performPing: Bool
+    let creationTimeMilliseconds: Int64 // Cache coding only
+
+    // For unit tests
+    init(display: Bool, performPing: Bool) {
+        self.display = display
+        self.performPing = performPing
+        creationTimeMilliseconds = Date().millisecondsSince1970
+    }
```

Comment:
Can this init be declared in an extension in the unit tests target?

Reply:
Hmm. That's odd. I think I tried but I got some error but now it seems to work.
Ok then I'm definitely putting this into an extension in test target

Reply:
Ah this init is used by another class in the SDK after all. I'm leaving it then. I'll remove the comment that it's for unit test only